<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MyQGLWidget" name="qglwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_view">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>view</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>scale</string>
     </property>
    </widget>
    <widget class="QSlider" name="SliderScale">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>rotate</string>
     </property>
    </widget>
    <widget class="QSlider" name="SliderRotate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="SliderTrans">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>trans</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_light">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>light</string>
    </property>
    <widget class="QRadioButton" name="radioButton1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_cow">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>cow</string>
    </property>
    <widget class="QSlider" name="SliderSpeed">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>speed</string>
     </property>
    </widget>
    <widget class="QSlider" name="SliderRadius">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>radius</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyQGLWidget</class>
   <extends>QWidget</extends>
   <header>myqglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
